### DISCUSSION TITLE
3004.388.6_2-gnuton0_alpha1

### DISCUSSION THREAD

**Discussion by gnuton** [gnuton/asuswrt-merlin.ng#549] [Announcements]
##  GNUton's Asus Merlin changelog  ##

Date: March 08, 2024
Release: 3004.388.6_2-gnuton1

  - ADDED: Support for TUF-AX3000_V2

<hr /><em>This discussion was created from the release <a href='https://github.com/gnuton/asuswrt-merlin.ng/releases/tag/3004.388.6_2-gnuton0_alpha1'>3004.388.6_2-gnuton0_alpha1</a>.</em>

**Comment by gnuton** [comment]
Please share your feedbacks here. Any feedback si welcome! I hope you enjoy this new pre-release with TUF-AX3000 V2. support

**Reply by luanhouse** [reply]
Thanks gnuton! 
I'm a very beginner with routers, I just flashed this version to my AX300_V2 and installed Astrill VPN on it, but is this correct when i select to astrill page, its shows blank.

screen shoot:
![image](https://github.com/gnuton/asuswrt-merlin.ng/assets/20237684/04e3f231-ef31-43b7-ab0a-6e22c63c7dd4)

astrill blank page:
![image](https://github.com/gnuton/asuswrt-merlin.ng/assets/20237684/6f0c0372-c1ae-4a2f-990f-03761c83ca51)

I just trying to bulid VPN for my family network by this way, is this would be fix (or implement) in future updates? 

Thanks again for your sharing!
Shuaiqi.

**Reply by gnuton** [reply]
Hi @luanhouse if you press F12 you can see if there is any error in the Javascript console.
if the page is blank there is something wrong. I have never installed this plugin so i do not have any clue about what's wrong.

**Reply by luanhouse** [reply]
There's error in console page.
![image](https://github.com/gnuton/asuswrt-merlin.ng/assets/20237684/c20180f5-fc53-4d64-836b-aaf615e0c785)

And I found that the link contents in source code is available to open and change the settings of Astrill.
http://192.168.50.1:8998/user/cgi-bin/astrill.cgi?token=wPlpHAgWQanCNLjBc8yeK0OojpAVL1EM
![image](https://github.com/gnuton/asuswrt-merlin.ng/assets/20237684/92e899dc-a383-4a8f-8f22-6fb272b8f9d8)

Where I can see from my astrill dealer website, AX-3000_V2 is not in the list, is that might the root cause? But I'm not sure of this.
https://www.astrill.com/wiki/Astrill_Setup_Manual:Getting_started_with_ASUS_Merlin_firmware_for_routers

If you are interested in astrill case, i'm glad to provide my own SSH for you to test.

Thank you!
Shuaiqi.

**Reply by viratsoft** [reply]
Traffic Analyzer - Statistic Not show data.
version:3004.388.6_2
GT-AX11000
Working fine After downgrade to 3004.388.6
Please solve this issue..

**Reply by gnuton** [reply]
you should debug what's the problem in content.js.

**Reply by gnuton** [reply]
not sure if that would help but you can install the image without TUF UI, not sure if that would help, but it's closer to standard merin.

**Comment by viratsoft** [comment]
Traffic Analyzer - Statistic Not show data.
version:3004.388.6_2
GT-AX11000

**Reply by shiang-luong** [reply]
I noticed my guest network loses access to the wan/Internet more often.  Just wondering if anyone else noticed that.  Also what data can I provide next time if it happens again?

**Reply by gnuton** [reply]
@shiang-luong you need to debug what happens on client side
